### ISSUE TITLE
Failed to open a new tab [10.0.2]

### ISSUE THREAD

**Issue by neuronetio** [cypress-io/cypress#22150]
### Current behavior

When I run headed chrome test `./node_modules/.bin/cypress run --config video=false --browser chrome --headed
` I got following error:
```js
Failed to open a new tab
Error: Failed to open a new tab
    at /*...*//.cache/Cypress/10.0.2/Cypress/resources/app/node_modules/chrome-remote-interface/lib/chrome.js:94:35
```

### Desired behavior

open each test file in separate window (not tab) because after each test browser window is closed and doesn't exists anymore

### Test code to reproduce

any code

### Cypress Version

10.0.2

**Comment by lmiller1990** [comment]
Hi, thanks for the bug report. Assuming this is only happening as of Cypress 10?

Can you share some information about your environment - OS, Node.js version, etc? I'm not able to reproduce this behaviour in any OS (tried Ubuntu, Windows, macOS).

**Comment by neuronetio** [comment]
Hi,
yes this is only happening in cypress 10
OS: Kubuntu 21.10
Node: v17.9.0
npm: 8.10.0

I had cypress 9.* and after migration process cypress stops working

**Comment by lmiller1990** [comment]
Potentially related: https://github.com/cypress-io/cypress/pull/22168

**Comment by mjhenkes** [comment]
@neuronetio, we have released cypress 10.1.0 with a fix for the possibly related issue #22168, could you try to re-create this issue again to see if we possibly fixed this issue?

**Comment by neuronetio** [comment]
In version `10.1.0` the problem has been resolved.
